variables.tf

```
@@ -248,6 +253,29 @@ variable "encryption_at_host_enabled" {
   default = true
 }
 
+variable "entra_id_login" {
+  description = "Configuration for enabling EntraID-based login for virtual machines. Set 'enabled' to true to activate the feature and specify the list of 'principal_ids' that are allowed to log in. Note: This feature requires at least 1GB of memory, and certain SKUs like 'Standard_B1ls', 'Basic_A0', and 'Standard_A0' are not supported."
```

Improve multiline comments, Markdown formatting, a little the text content:

```suggestion
  description = <<-EOD
    Configures Entra ID-based login for virtual machines. Set `enabled` to `true` to activate this feature and define the list of `principal_ids` permitted to log in.  

    **Note**: This feature requires at least 1GB of memory and is not supported on certain SKUs, including `Standard_B1ls`, `Basic_A0`, and `Standard_A0`.
  EOD
```